xapian-core/include/xapian/weight.h

```diff
@@ -409,6 +409,11 @@ class XAPIAN_VISIBILITY_DEFAULT TfIdfWeight : public Weight {
     /// The factor to multiply with the weight.
     double factor;
 
+    /// Parameters slope and delta in the Piv+ normalization weighting formula.
+    /// Default values: 0.2 and 1.0 respectively.
+    double param_slope = 0.2;
+    double param_delta = 1.0;
+
     TfIdfWeight * clone() const;
```

Comment:
It's probably most useful if I interactively help you work through how to interpret these.  There's a bit of a knack to working out what the problem is, and if you understand the processes to go through then you're more likely to be able to work it out yourself next time.

I'd suggest we do that on IRC.  I'm about to disconnect for the evening, but I'm mostly around NZ daytime currently - just ping me on #xapian.